dt-reports/magic-url-endpoints.php

```diff
@@ -121,7 +132,24 @@ public function email_magic( WP_REST_Request $request ) {
             }
 
             // check if email exists to send to
-            if ( ! isset( $post_record['contact_email'][0] ) && empty( $params['email'] ) ) {
+            $emails = [];
+            if ( isset( $params['email'] ) && ! empty( $params['email'] ) ){
+                $emails[]   = $params['email'];
+            }
+            else if ( isset( $post_record['contact_email'][0] ) ) {
```

Comment:
If the single field for a contact "contact_email" field is available, go with the email found here. This will serve contacts.